<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0F151A;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>0</y>
       <width>741</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icon &amp; image/Mask group.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dangnhap">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>370</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#CC0033;
border: 1px solid white;
border-radius: 12px;
color:white;</string>
     </property>
     <property name="text">
      <string>Đăng nhập</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ten_tk">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>280</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#fff;
border: 1px solid white;
color:black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_mat_khau">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>320</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#fff;
border: 1px solid white;
color:black;
border-right:none;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>450</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: black;color:red;
padding-left:2px;</string>
     </property>
     <property name="text">
      <string>KHUYẾN CÁO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>480</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: black;color:white;</string>
     </property>
     <property name="text">
      <string>Không chơi quá 180 phút</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>450</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: black;color:red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icon &amp; image/299051_ban_sign_icon.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border: 1px solid white;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Icon &amp; image/352000_account_box_icon.ico</normaloff>../Icon &amp; image/352000_account_box_icon.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>39</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border: 1px solid white;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Icon &amp; image/3592656_lock_locked_padlock_password_privacy_icon.ico</normaloff>../Icon &amp; image/3592656_lock_locked_padlock_password_privacy_icon.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>29</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hien_thi_mk">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#fff;
border: 1px solid white;
color:white;
border-left:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Icon &amp; image/7787574_blind_eye_view_vision_magnifier_icon.ico</normaloff>../Icon &amp; image/7787574_blind_eye_view_vision_magnifier_icon.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_chuyen">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CC0033;
border: 1px solid white;
border-radius: 12px;
color:white;</string>
    </property>
    <property name="text">
     <string>Cyber Game QDHQ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1471</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
